packages/server/src/tests/banners/channelBannerTests.ts

```diff
@@ -60,12 +43,6 @@ export const BannerTemplateComponentTypes: {
     signIn: 'ACQUISITIONS_ENGAGEMENT_BANNER',
 };
 
-export const BannerTemplateProducts: {
-    [key in BannerTemplate]?: OphanProduct[];
-} = {
-    [BannerTemplate.GuardianWeeklyBanner]: ['PRINT_SUBSCRIPTION'],
```

Comment:
setting the `product` in the tracking was only relevant for the GuardianWeeklyBanner, and given we've not used this banner in a while I think it's safe to say it's not needed anymore